internal/backup_list_handler_test.go

```diff
@@ -194,3 +194,87 @@ func TestWritePrettyBackupList_EmptyColumnsValues(t *testing.T) {
 
 	assert.Equal(t, expectedRes, b.String())
 }
+
+func TestWriteBackupList_NoBackups(t *testing.T) {
+	expectedRes := "name last_modified wal_segment_backup_start\n"
+	backups := make([]internal.BackupTime, 0)
+
+	b := bytes.Buffer{}
+	internal.WriteBackupList(backups, &b)
+
+	assert.Equal(t, expectedRes, b.String())
+}
+
+func TestWriteBackupList_EmptyColumnsValues(t *testing.T) {
+	expectedRes := `name last_modified        wal_segment_backup_start
+     0001-01-01T00:00:00Z 
+b1   0001-01-01T00:00:00Z 
+     0001-01-01T00:00:00Z shortWallName0
+`
+	backups := []internal.BackupTime{
```

Comment:
This struct looks identical to line 178 (same goes for TestWriteBackupList_ShortColumnsValues and TestWriteBackupList_LongColumnsValues). I suggest extracting these structs to remove duplication.